test/int_processing_eras.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+// this file contains logic for processingeras API
+// the HTTP request body is defined by dbs.ProcessingEras struct defined in dbs/processingeras.go
+// the HTTP response body is defined by dbs.ProcessingEra struct defined in dbs/processingeras.go
+// the HTTP handlers and endpoints are defined in the EndpointTestCase struct defined in test/integration_cases.go
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/vkuznet/dbs2go/dbs"
+	"github.com/vkuznet/dbs2go/web"
+)
+
+// processingeras endpoint tests
+// TODO: Rest of test cases
+func getProcessingErasTestTable(t *testing.T) EndpointTestCase {
+	procErasReq := dbs.ProcessingEras{
+		PROCESSING_VERSION: int64(TestData.ProcessingVersion),
+		DESCRIPTION:        "this_is_a_test",
+		CREATE_BY:          "tester",
+	}
+	procErasResp := dbs.ProcessingEra{
```

Comment:
there is no `dbs.ProcessingEra` in current codebase, may be you added it in your fork and forget to add to this commit.